### PR TITLE
Add build status badge to README

### PR DESCRIPTION
**Pull request by csomh**
Signed-off-by: Hunor Csomortáni <csomh@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Overview
 
+[![Docker Repository on Quay](https://quay.io/repository/redhat-aqe/tox/status
```
Comment (sochotnicky) [inline README.md]: Container repository :-)

Comment (mmalina) [reply README.md]: I think it should be "Container image repository on Quauy" ;)

Comment (csomh) [reply README.md]: oh, common! :D

Comment (sochotnicky) [reply README.md]: Quay.io or just Quay - but not Quauy :-)

**Pull request merged**